osu.Game.Tests/Visual/SongSelect/TestSceneLeaderboard.cs

```diff
@@ -245,34 +241,75 @@ private void newScores()
             leaderboard.Scores = scores;
         }
 
-        private void realBeatmap()
+        private void rankedBeatmap()
         {
             leaderboard.Beatmap = new BeatmapInfo
             {
-                StarDifficulty = 1.36,
-                Version = @"BASIC",
                 OnlineBeatmapID = 1113057,
-                Ruleset = rulesets.GetRuleset(0),
-                BaseDifficulty = new BeatmapDifficulty
-                {
-                    CircleSize = 4,
-                    DrainRate = 6.5f,
-                    OverallDifficulty = 6.5f,
-                    ApproachRate = 5,
-                },
-                OnlineInfo = new BeatmapOnlineInfo
-                {
-                    Length = 115000,
-                    CircleCount = 265,
-                    SliderCount = 71,
-                    PlayCount = 47906,
-                    PassCount = 19899,
-                },
-                Metrics = new BeatmapMetrics
-                {
-                    Fails = Enumerable.Range(1, 100).Select(i => i % 12 - 6).ToArray(),
-                    Retries = Enumerable.Range(-2, 100).Select(i => i % 12 - 6).ToArray(),
-                },
+                Status = BeatmapSetOnlineStatus.Ranked,
+            };
+        }
+
+        private void approvedBeatmap()
```

Comment:
How about a single method that sets the Beatmap with a passed status? Would make it a lot smaller.

Reply:
This kinda still applies.